credentials/src/main/kotlin/SSI.kt

```diff
@@ -241,21 +242,18 @@ public class VerifiablePresentation private constructor() {
     @Throws(Exception::class)
     public fun create(signOptions: SignOptions, createVpOptions: CreateVpOptions): VpJwt {
 
-      val usableVcJwts: List<VcJwt> =
-        selectFrom(createVpOptions.presentationDefinition, createVpOptions.verifiableCredentialJwts)
-
-      if (usableVcJwts.size == 0) {
-        throw Exception("There are no useable Vcs that correspond to the presentation definition")
+      // Verify VC Validity
+      createVpOptions.verifiableCredentialJwts.forEach { vcJwt ->
+        VerifiableCredential.verify(vcJwt, createVpOptions.resolver)
```

Comment:
You aren't using the value returned by `verify`. Is that intentional?

Reply:
Good catch, I was thinking it worked like the web5-js version.

I think we should change it in the future to be similar to web5-js version, just throws exceptions and if no exception is thrown it is valid.

Will fix!